### PR TITLE
Api 26694 datamart ignoreproperties

### PR DESCRIPTION
**Pull request by ChrisBratti-VA**

### CODE REVIEW

File: facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartDetailedService.java [hunk 1]
```diff
@@ -122,6 +122,7 @@ public DatamartDetailedService serviceName(String serviceName) {
   @JsonInclude(value = JsonInclude.Include.NON_EMPTY, content = JsonInclude.Include.NON_EMPTY)
   @JsonPropertyOrder({"name", "serviceId", "serviceType"})
   @Schema(description = "Service information.")
+  @JsonIgnoreProperties(ignoreUnknown = true)
   public static final class ServiceInfo {
     @Schema(description = "Service id.", example = "covid19Vaccine")
     @JsonAlias({"service_id", "service_api_id"})
```
Comment (mmccollum-va) [inline facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartDetailedService.java:128]: Consider adding `@JsonIgnoreProperties(ignoreUnknown = true)` to `ServiceInfoBuilder` inner class

File: facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartFacility.java [hunk 2]
```diff
@@ -583,6 +588,7 @@ public FacilityAttributes.FacilityAttributesBuilder instructions(String val) {
   @Builder
   @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
   @JsonInclude(value = Include.NON_EMPTY, content = Include.NON_EMPTY)
+  @JsonIgnoreProperties(ignoreUnknown = true)
   public static final class Hours {
     String sunday;
```
Comment (mmccollum-va) [inline facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartFacility.java:594]: Consider  adding `@JsonIgnoreProperties(ignoreUnknown = true)` to `HoursBuilder` inner class

File: facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartFacility.java [hunk 3]
```diff
@@ -492,6 +496,7 @@ public static final class Addresses {
   @Builder
   @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
   @JsonInclude(value = Include.NON_EMPTY, content = Include.NON_EMPTY)
+  @JsonIgnoreProperties(ignoreUnknown = true)
   @JsonPropertyOrder({
     "name",
     "facility_type",
```
Comment (mmccollum-va) [inline facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartFacility.java:502]: Consider adding `@JsonIgnoreProperties(ignoreUnknown = true)` to `FacilitiesAttributesBuilder` inner class

File: facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartFacility.java [hunk 4]
```diff
@@ -759,6 +772,7 @@ public static final class Services {
   @Builder
   @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
   @JsonInclude(value = Include.NON_EMPTY, content = Include.NON_EMPTY)
+  @JsonIgnoreProperties(ignoreUnknown = true)
   @JsonPropertyOrder({"name", "serviceId"})
   @AllArgsConstructor
   public static final class Service<T extends TypedService> implements Comparable<Service<T>> {
```
Comment (mmccollum-va) [inline facilities/src/main/java/gov/va/api/lighthouse/facilities/DatamartFacility.java:778]: Consider adding `@JsonIgnoreProperties(ignoreUnknown = true)` to `ServiceBuilder` inner class

**Review by mmccollum-va — CHANGES_REQUESTED** [review]

**Review by as-va — CHANGES_REQUESTED** [review]
Can you add into the DeserializerTest a verification that receiving an unknown field is properly ignored and still results in a valid object build from the test data?

**Review by as-va — APPROVED** [review]

**Pull request merged**